config/peerpods/podvm/aws-VM-image-create-job.yaml

```diff
@@ -59,6 +57,7 @@ spec:
         - -c
         - |
           set -e
+          [[ ! "${IMAGE_NAME}" ]] && UUID=$(uuidgen) && export IMAGE_NAME="peer-pod-ami-${UUID::6}" && echo "IMAGE_NAME:${IMAGE_NAME}"
```

Comment:
If we are not requiring IMAGE_NAME to be explicitly unset we probably need to add that to the docs.

Reply:
User shouldn't even know it's possible to set it, this variable is not mentioned in the docs